espflash/src/targets/esp32c2.rs

```diff
@@ -108,43 +94,6 @@ impl Target for Esp32c2 {
         HashMap::from(encodings)
     }
 
-    fn flash_image<'a>(
-        &self,
-        elf_data: &'a [u8],
-        flash_data: FlashData,
-        _chip_revision: Option<(u32, u32)>,
```

Comment:
The current `flash_image` does not take/use chip revision as previously it was passed for the `flash` subcommand but then it wasn't used at all.